packages/terra-date-input/tests/wdio/date-input-spec.js

```diff
@@ -336,7 +336,7 @@ Terra.describeViewports('Date Input', ['medium'], () => {
       Terra.it.matchesScreenshot();
     });
 
```

Comment:
Can we add a test to test clearing the month by selecting the placeholder?

Reply:
Added test 6f09098